src/SwissPaymentSlip/SwissPaymentSlipFpdf/PaymentSlipFpdf.php

```diff
@@ -24,32 +26,37 @@ class PaymentSlipFpdf extends PaymentSlipPdf
 {
 	protected $rgbColors = array();
 
-	/**
-	 * The PDF engine object to generate the PDF output
-	 *
-	 * @var null|FPDF The PDF engine object
-	 */
+    /**
+     * The PDF engine object to generate the PDF output with
+     *
+     * @var null|FPDF The PDF engine object
+     */
 	protected $pdfEngine = null;
 
-	/**
-	 * @var string;
-	 */
+    /**
```

Comment:
All the doc blocks seem to be intended wrongly.
What were your settings?
By PSR-2 standard it should be 4 spaces.